XLS-0065d-single-asset-vault/README.md

```diff
@@ -0,0 +1,653 @@
+
+###### 2.1.6.2.1 `MPToken` Values
+
+The `MPToken` values should be set as per the `MPT` [specification](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0033d-multi-purpose-tokens#2122-fields).
+
+| **Condition**     | **Transferable**   | **Non-Transferable** |
+| ----------------- | ------------------ | -------------------- |
+| **Public Vault**  | No Flags           | `lsfMPTAuthorized`   |
+| **Private Vault** | `lsfMPTAuthorized` | `lsfMPTAuthorized`   |
+
+#### 2.1.7 Exchange Algorithm
+
+Exchange Algorithm refers to the logic that is used to exchange assets into shares and shares into assets. This logic is executed when depositing or redeeming liquidity. A Vault comes with the default exchange algorithm, which is detailed below.
+
+##### 2.1.7.1 Unrealized Loss
+
+A well-informed depositor may learn of an incoming loss and redeem their shares early, causing the remaining depositors to bear the full loss. To discourage such behaviour, we introduce a concept of "paper loss," captured by the `Vault` object's `LossUnrealized` attribute. The "paper loss" captures a potential loss the vault may experience and thus temporarily decreases the vault value. Only a protocol connected to the `Vault` may increase or decrease the `LossUnrealized` attribute.
+
+The "paper loss" temporarily decreases the vault value. A malicious depositor may take advantage of this to deposit assets at a lowered price and withdraw them once the price increases.
+
+Consider a vault with a total value of $1.0m and total shares of $1.0m. Assume the "paper loss" for the vault is $900k. The new exchange rate is as follows:
+
+$$
+exchangeRate = \frac{AssetsTotal - LossUnrealized}{SharesTotal}
+$$
+
+$$
+exchangeRate = \frac{1,000,000 - 900,000}{1,000,000} = 0.1
+$$
+
+After the "paper loss" is cleared, the new effective exchange rate would be as follows:
+
+$$
+exchangeRate = \frac{AssetsTotal}{SharesTotal}
+$$
+
+$$
+exchangeRate = \frac{1,000,000}{1,000,000} = 1.0
+$$
+
+A depositor could deposit $100k assets at a 0.1 exchange rate and get 1.0m shares. Once the "paper loss" is cleared, their shares would be worth $1.0m.
+
+To account for this problem, the Vault must use two different exchange rate models: one for depositing assets and one for withdrawing them.
+
+##### 2.1.7.2 Exchange Rate Algorithms
+
+**Variables**
+
+The following variables define the Vault balance:
+
+- $\Gamma_{share}$ - the total number of shares issued by the vault.
+- $\Gamma_{asset}$ - the total assets in the vault, including any future yield.
+
+- $\Delta_{asset}$ - the change in the total amount of assets after a deposit, withdrawal, or redemption.
+- $\Delta_{share}$ - che change in the total amount of shares after a deposit, withdrawal, or redemption.
+
+- $\iota$ - the unrealized loss of the vault.
```

Comment:
Do you want to make this a capital letter like the others?

Reply:
Fair point.

Reply:
It seems that Github LateX renderer does not support uppercase iota. I guess for now, I'll keep it lower-case.